### ISSUE TITLE
Expanded view in Missed Subtitle Management partially broken

### ISSUE THREAD

**Issue by medariox** [SiCKRAGETV/sickrage-issues#2987]
Manage > Missed Subtitle Management > Any Language [Manage] > Expand

Since a picture says more than a thousand words:
![expanded view](https://snapr.pw/i/7b143d24d1.png)

I'm on master.

**Comment by neoatomic** [comment]
Confirmed on `BRANCH: (develop) / COMMIT: (62e67b759534257cb606eb811192a6745fa18e89) `

![naamloos](https://cloud.githubusercontent.com/assets/7928052/10131797/26bd55c4-65d3-11e5-89c8-098595289a68.png)

**Comment by Kumolo** [comment]
@medariox @neoatomic: Just noticed I currently (master, f913285d1decce196c6054c43507582fb2a2a7ee) cant't figure how to manually download subs for a single tv episode. Wasn't there sort of a "CC" button next to the "Search" button in the Show View? Is the view missing or am I just lost?

![foo](https://cloud.githubusercontent.com/assets/12941715/10173790/8241f0c6-66e6-11e5-9954-e11e4837a404.JPG)

**Comment by duramato** [comment]
@Kumolo select column bottun check subtitles

**Comment by Kumolo** [comment]
@duramato Thanks, but that is indeed activated  :-( 

![unbenannt](https://cloud.githubusercontent.com/assets/12941715/10177425/74598032-66fa-11e5-8365-effcdeedbaee.JPG)

**Comment by fernandog** [comment]
@OmgImAlexis do you know what's wrong?

**Comment by OmgImAlexis** [comment]
@fernandog it may be from one of the PRs you submitted a few days ago where you needed to change some of the sorting vars. I'll look into it.

**Comment by fernandog** [comment]
ok! thanks

**Comment by OmgImAlexis** [comment]
Since I don't use/have any subtitles would someone be able to send me their `sickbeard.db`? Email is on my github profile.

**Comment by fernandog** [comment]
create a empty file with the same name as the movie file. rename mkv to srt

**Comment by miigotu** [comment]
the mkv can even be a fake file.

touch somefile.mkv somefile.srt

Add 2 languages to the languages selection and select a provider.

**Comment by OmgImAlexis** [comment]
Does anyone know how to subtitles page use to look? I can't find anything to do with subtitles that broke it. I know the flag images are kind of broken but the layout should still work fine.

**Comment by OmgImAlexis** [comment]
@Kumolo I have no clue why it's not showing on yours..
![screen shot 2015-10-02 at 2 24 17 pm](https://cloud.githubusercontent.com/assets/6525926/10239742/4ef922e8-6911-11e5-9fc0-a1f5b9ff38af.png)

**Comment by Kumolo** [comment]
@OmgImAlexis Strange. Don't know what is wrong. However, for the last days, everyday I get an ERROR in Log saying "Could not refresh subtitles". The enry is always from "04:08:54 SHOWQUEUE-REFRESH"

Here are my subs settings:
[Subtitles]
use_subtitles = 1
subtitles_languages = en
SUBTITLES_SERVICES_LIST = "addic7ed,opensubtitles,tvsubtitles,itasa,subtitulos,usub,subscenter,thesubdb,subswiki"
SUBTITLES_SERVICES_ENABLED = 1|1|1|0|0|0|0|0|0
subtitles_dir = /share/Download/sickrage/subs
subtitles_default = 1
subtitles_history = 1
embedded_subtitles_all = 0
subtitles_finder_frequency = 8
subtitles_multi = 1
subtitles_extra_scripts = ""

Any idea how we can trace the root cause?

**Comment by OmgImAlexis** [comment]
@Kumolo my email is on my github profile, if you want to add a username and password to SR and send me your IP I can take a look, you'll just need to port forward.

**Comment by miigotu** [comment]
Since the issue that this #2987 is actually about is fixed......

**Comment by Kumolo** [comment]
@OmgImAlexis Thanks for the offer to help. Meanwhile, I guess I figured out what was wrong. I noticed that my subs settings in config file was

Subtitles]
use_subtitles = 1
subtitles_languages = **en**

while in the GUI, subtitle languages was empty. I chose "English" and restarted SR. I noticed that now the config file says "eng" not "en". After this restart, none of my previously selected subs providers was selected. After selecting them and restart, I think I am back to normal.
